PWGCF/Flow/Tasks/flowTask.cxx

```diff
@@ -248,6 +257,27 @@ struct FlowTask {
       fWeights->Init(true, false);
     }
 
+    TList* groupNUAWeightlist = new TList();
+    groupNUAWeightlist->SetOwner(true);
+    fGroupNUAList.setObject(groupNUAWeightlist);
+
+    if (cfgOutputGroupNUAWeights) {
+      groupNUAWeightPtr.resize(cfgGroupSplitRunNumber.value.size() + 1);
+      for (uint i = 0; i < cfgGroupSplitRunNumber.value.size() + 1; i++) {
+        GFWWeights* groupweight = nullptr;
+        if (i < cfgGroupSplitRunNumber.value.size())
+          groupweight = new GFWWeights(Form("groupweight_%d", cfgGroupSplitRunNumber.value[i]));
+        else
+          groupweight = new GFWWeights(Form("groupweight_last"));
```

Comment:
Be aware that in this case, all objects are incorporated to the list with the same name